Readme.md

```diff
@@ -2,260 +2,308 @@
 
 [![Gitter](https://badges.gitter.im/rust-sgx-sdk/community.svg)](https://gitter.im/rust-sgx-sdk/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-# Rust SGX SDK
-Rust SGX SDK helps developers write Intel SGX applications in Rust programming language. [[CCS'17 Paper pdf]](documents/ccsp17.pdf) [[CCS'19 Paper pdf]](https://dingelish.com/ccs19.pdf)
+# Apache Teaclave Rust-SGX SDK
```

Comment:
We can formally call it **Teaclave SGX SDK** in all document.

In the introduction, We can say that the project is also called Rust SGX SDK.